bazel/README.md

```diff
@@ -116,6 +116,13 @@ bazel test -c dbg //test/server:backtrace_test
 You will need to use either a `dbg` build type or the `--define
 debug_symbols=yes` option to get symbol information in the binaries.
 
+By default main.cc will install signal handlers to print backtraces at the
+location where a fatal signal occurred.  The signal handler will re-raise the
+fatal signal with the default handler so a core file will still be dumped after
+the stack trace is logged.  To inhibit this behavior use
+`--define=signal_trace=disabled` on the Bazel command line. No signal handlers will
```

Comment:
You might want to set this in the `tools/bazel.rc` for the ASAN runs. I'm not sure where ASAN installs its signal handlers (based on your experiences with the tests, it seems after this), but this might be safer in general to avoid conflict.